python/lsst/ts/observatory/control/auxtel/atcs.py

```diff
@@ -267,11 +268,12 @@ async def slew_dome_to(self, az, check=None):
                 "Make sure it is clear to operate the dome before doing so."
             )
 
-        self.log.warning(
-            "Sending ATDomeTrajectory to DISABED state. Component will be left in DISABLED"
-            "state or else it may send the ATDome back to alignment with the telescope."
-        )
-        await salobj.set_summary_state(self.rem.atdometrajectory, salobj.State.DISABLED)
+        if self.check.atdometrajectory:
+            self.log.debug("Disable atdometrajectory following.")
+
+            await self.rem.atdometrajectory.cmd_setFollowingMode.set_start(
+                enable=False, timeout=self.fast_timeout
+            )
```

Comment:
I see these exact lines of code repeated several times in this file. Please consider extracting them to a function and call the function where needed.

Reply:
thanks for the suggestion! adopted!